doc/LOCATIONS.md

```diff
@@ -0,0 +1,155 @@
+# API Documentation for /Locations End Point
+
+## Show all locations
+
+**URL :** /locations
+
+**Method :** GET
+
+**URL Params :** -
+
+**Data Params :** -
+
+**Success Response:** Code: 200, Content: 
+
+    [
+        {
+            "id": 1,
+            "name": "Dnipro",
+            "country": {
+                "id": 1,
+                "name": "Ukraine"
+            }
+        },
+        {
+            "id": 2,
+            "name": "Sofia",
+            "country": {
+                "id": 2,
+                "name": "Hungary"
+            }
+        }
+    ]
+
+**Error Response :** Code: 401 Unauthorized
+
+## Show all the groups, that belong to the specified location by location Id.
+
+**URL :** /locations/{locationIds}/groups
+
+**Method :** GET
+
+**URL Params :** Required: `{locationIds}`(type: integer), example: `2`
```

Comment:
Will it be any better if we also construct full url in example? Like this: "/locations/2/groups"